### ISSUE TITLE
For training

### ISSUE THREAD

**Issue by JessieSun-lalala** [MCG-NJU/PDPP#2]
Thanks for your work and sharing. When I use a custom dataset to train PDPP, the loss increases sharply around 10epoch. I tried adjusting the learning rate but it didn't work. Have you ever encountered this problem? Looking forward to your reply

**Comment by pPetrichor** [comment]
Thanks for your question. Since we don't know the specific content of your dataset, we can't give a definite solution. However, reducing the learning rate and reasonably applying the warm up strategy should be some methods to solve the problem of sudden increase in training loss.
